reporthandling/results/v1/reportsummary/datastructures.go

```diff
@@ -14,7 +14,7 @@ type ControlSummaries map[string]ControlSummary
 
 // SummaryDetails detailed summary of the scanning. will contain versions, counters, etc.
 type SummaryDetails struct {
-	SeverityCounters ISeverityCounters   `json:"severityCounters,omitempty"`
+	SeverityCounters SeverityCounters   `json:"severityCounters,omitempty"`
```

Comment:
As we have previously discussed, I changed this to be a concrete struct, so we can serialize to JSON conveniently.

The alternative is to keep the interface object and use a custom `MarshalJSON()` method, but we are sorta rushing.